sdk/python/kfp/_client.py

```diff
@@ -741,6 +741,28 @@ def upload_pipeline(
       IPython.display.display(IPython.display.HTML(html))
     return response
 
+  def upload_pipeline_version(
+    self,
+    pipeline_package_path,
+    pipeline_name: str,
+    pipeline_id: str,
+  ):
+    """Uploads a new version of the pipeline to the Kubeflow Pipelines cluster.
+    Args:
+      pipeline_package_path: Local path to the pipeline package.
+      pipeline_name:  Name of the pipeline to be shown in the UI.
+      pipeline_id: Id of the pipeline.
+    Returns:
+      Server response object containing pipleine id and other information.
+    """
+
+    response = self._upload_api.upload_pipeline_version(pipeline_package_path, name=pipeline_name, pipelineid=pipeline_id)
```

Comment:
Can we handle errors?
e.g. use a non existent pipeline_id.

Reply:
If I look at the other functions in the same file, (for example `upload_pipeline`, they also don't have additional error handling, do you want to add it for this one anyway?